src/govuk/overrides/_display.scss

```diff
@@ -19,4 +19,10 @@
   .govuk-\!-display-none {
     display: none !important;
   }
+
+  @include govuk-media-query($media-type: print) {
+    .govuk-\!-print-display-none {
```

Comment:
Sorry for this after approving things.

Can we be careful to consider this naming vs all the other overrides?

Perhaps this should be `.govuk-\!-display-none-print` as the format seems to be `.govuk-\!-<PROPERTY>-<VALUE/SCALE-MODIFIER>`

Reply:
I've pinged the team on this...

Reply:
We put print first so that it was more easily differentiated from `govuk-!-display-none`.

`print-display-none` matches the order in the stylesheet:
```
@media print {
  .blah {
    display: none
  }
}
```

Reply:
@36degrees @hannalaakso thoughts?

Reply:
As far as I can see, we don't have any other overrides that are conditional on media queries. Probably the closest existing class would be e.g. `govuk-grid-column-one-third-from-desktop`, where the 'media query' part is last?

If we did go with `govuk-`!-display-none-print` that means it would also match if you searched your code base for `govuk-\!-display-none`, but that's also true for `govuk-grid-column-one-third-from-desktop` and `govuk-grid-column-one-third` and I'm not aware of that having been an issue for anyone.